jar-connector/src/main/java/com/sixsq/slipstream/connector/SystemConfigurationParametersFactoryBase.java

```diff
@@ -123,4 +136,14 @@ protected void putMandatoryMaxIaasWorkers() throws ValidationException {
 				RuntimeParameter.MAX_JAAS_WORKERS_DEFAULT);
 	}
 
+	protected void putMandatoryContextualizationType() throws ValidationException {
+		putMandatoryEnumParameter(
+				super.constructKey(NATIVE_CONTEXTUALIZATION_KEY),
+				"Use native cloud contextualisation",
+				getNativeContextualizationOptions(),
+				NATIVE_CONTEXTUALIZATION_DEFAULT,
+				"Here you can define when SlipStream should use the native Cloud contextualization mechanisme or when it should try other methods like SSH and WinRM. <br/>",
```

Comment:
Minor.  Typo in `mechanisme` -> `mechanism`.  Suggestion: remove `mechanisme` word at all.  Because later it's called `method` anyways.